picamera2/picamera2.py

```diff
@@ -128,7 +129,7 @@ def asynchronous(self) -> bool:
 
     @property
     def camera_properties(self) -> dict:
-        return {} if self.camera is None else self.camera.properties
+        return {} if self.camera is None else self.properties
```

Comment:
I think it would be more usual to use `self.camera_properties_` rather than `self.properties`.